### PR TITLE
Editing notebooks to prepare for DevSite import

### PR DESCRIPTION
**Pull request by rszper**
Editing RunInference notebooks to prepare for DevSite import

R: @damccorm 
R: @AnandInguva 

------------------------

Thank you for your contribution! Follow this checklist to help us incorporate your contribution quickly and easily:

 - [ ] Mention the appropriate issue in your description (for example: `addresses #123`), if applicable. This will automatically add a link to the pull request in the issue. If you would like the issue to automatically close on merging the pull request, comment `fixes #<ISSUE NUMBER>` instead.
 - [ ] Update `CHANGES.md` with noteworthy changes.
 - [ ] If this contribution is large, please file an Apache [Individual Contributor License Agreement](https://www.apache.org/licenses/icla.pdf).

See the [Contributor Guide](https://beam.apache.org/contribute) for more tips on [how to make review process smoother](https://github.com/apache/beam/blob/master/CONTRIBUTING.md#make-the-reviewers-job-easier).

To check the build health, please visit [https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md](https://github.com/apache/beam/blob/master/.test-infra/BUILD_STATUS.md)

GitHub Actions Tests Status (on master branch)
------------------------------------------------------------------------------------------------
[Build python source distribution and wheels](https://github.com/apache/beam/actions?query=workflow%3A%22Build+python+source+distribution+and+wheels%22+branch%3Amaster+event%3Aschedule)
[Python tests](https://github.com/apache/beam/actions?query=workflow%3A%22Python+Tests%22+branch%3Amaster+event%3Aschedule)
[Java tests](https://github.com/apache/beam/actions?query=workflow%3A%22Java+Tests%22+branch%3Amaster+event%3Aschedule)
[Go tests](https://github.com/apache/beam/actions?query=workflow%3A%22Go+tests%22+branch%3Amaster+event%3Aschedule)

See [CI.md](https://github.com/apache/beam/blob/master/CI.md) for more information about GitHub Actions CI or the [workflows README](https://github.com/apache/beam/blob/master/.github/workflows/README.md) to see a list of phrases to trigger workflows.

### CODE REVIEW

File: examples/notebooks/beam-ml/per_key_models.ipynb [hunk 1]
```diff
@@ -107,12 +107,7 @@
         }
       ],
       "source": [
-        "# Note that this notebook currently installs from Beam head since this feature hasn't been released yet.\n",
-        "# It will be released with version 2.51.0, at which point you can install with the following command:\n",
-        "# !pip install apache_beam[gcp]>=2.51.0 --quiet\n",
-        "!git clone https://github.com/apache/beam\n",
-        "!pip install -r beam/sdks/python/build-requirements.txt\n",
-        "!pip install -e ./beam/sdks/python[gcp]\n",
+        "!pip install apache_beam[gcp]>=2.51.0 --quiet\n",
```
Comment (rszper) [inline examples/notebooks/beam-ml/per_key_models.ipynb:110]: @AnandInguva @damccorm I'm planning to merge this after the 2.51.0 release, so this section needs to be updated, but please check to see if I updated it correctly. Thank you!

**Comment by rszper** [comment]
R: @damccorm
R: @AnandInguva

File: examples/notebooks/beam-ml/per_key_models.ipynb (hunk 1 above)
Comment (damccorm) [reply examples/notebooks/beam-ml/per_key_models.ipynb:110]: Ah, I actually had a pr open to do this already (and it is now merged - #28801)

**Review by damccorm — APPROVED** [review]
Thanks!

**Pull request merged**